data/856/720/45/85672045.geojson

```diff
@@ -309,8 +318,9 @@
         "quattroshapes"
     ],
     "src:lbl:centroid":"mapshaper",
-    "src:population":"statoids",
+    "src:population":"wk",
     "src:population:date":"2010-05-17",
+    "src:population_date":"2020-01-01",
```

Comment:
> "src:population:date":"2010-05-17",
>  "src:population_date":"2020-01-01",

Only one of these should be true... so delete `src:population:date` from this PR for this and other relevant records please.

Reply:
Will take care of this in an upcoming commit...